src/DataReader/mod.rs

```diff
@@ -3,6 +3,7 @@ pub mod json_reader;
 pub mod json_writer;
 pub mod parse_edge;
 pub mod parse_invariant;
+#[allow(deprecated)]
```

Comment:
pls no

Reply:
yes #HateClippy

Reply:
It was a suggestion by Sebastian to temporarily fix clippy since no one in this project will be fixing it either way.